22_pipeslab/pipeslab.txt

```diff
@@ -31,4 +55,15 @@ What was printed? Why? Fix the command line to get the output:
 2 Dec
 2 Nov
 
+who | tr -s " " | cut -d ' ' -f 3| cut -f2 -d" "| cut -f2 -d-| uniq -c
+
+[Note: Below is the ouput I am getting on my machine,
+We can use the tr command to replace for example 05 of May but maybe there is a better why to do it?]  
+
```

Comment:
This is cause by locale definitions which indicate to `who` you prefer to get its month name as a number. You can fix it with: `LC_ALL=C who | tr -s ' ' | cut -d ' ' -f 3 | sort | uniq -c`